xknx/io/transport/udp_transport.py

```diff
@@ -72,10 +76,16 @@ def __init__(
         self.multicast = multicast
 
         self.callbacks = []
+        self.local_addr_assigned: tuple[str, int] | None = None
         self.transport: asyncio.DatagramTransport | None = None
+        self.mcast_transport: asyncio.DatagramTransport | None = None
 
     def data_received_callback(self, raw: bytes, source: tuple[str, int]) -> None:
         """Parse and process KNXIP frame. Callback for having received an UDP packet."""
+        if self.multicast and source == self.local_addr_assigned:
+            knx_logger.debug("Received echoed KNXIPFrame")
```

Comment:
```suggestion
            raw_socket_logger.debug("Discarded echoed datagram")
```
I think this is better handled by `raw_socket_logger` since we haven't parsed it to a KNXIPFrame yet.